source/_integrations/rfxtrx.markdown

```diff
@@ -344,11 +344,23 @@ action:
     event: 0b1111e003af16aa10000060
 ```
 
+Alternatively:
+
+- Go to home-assistant-IP:8123/developer-tools/service.
+- Select: `RFXCOM RFXtrx: Send` from the Service drop-down menu.
```

Comment:
Please link directly to the service using a [My Home Assistant](https://my.home-assistant.io/create-link/) link